open-weather-connector-product/README.md

```diff
@@ -0,0 +1,40 @@
+# Open Weather Connector
```

Comment:
If I am not wrong, the correct way to write it is: OpenWeather (no space). Can you please make sure to make these changes overall?